rfc/rfc009-verifiable-presentations.md

```diff
@@ -92,308 +34,21 @@ The terminology used is defined in https://www.w3.org/TR/did-core/#terminology a
-    }
-},
-"signature": "<signature of the holder>"
-
-```
-
-Date and time values MUST be a number containing a NumericDate value (Unix epoch timestamps).
-
-
-
-## 6. Realization of the verifiable credential
-
-The JWT of the credential.
-
-```json
-"protected": {
-    "alg": "EdDSA", 
-    "kid": "<reference to assertion method in did document>",
-    "typ": "JWT"
-},
-"payload": {
-    "jti": "<credential id>",
-    "iss": "<did of issuer>",
- 		"sub": "<did of holder>",
-    "nbf": "<valid from date, same as issuance date or later>",
-    "exp": "<expiration date>",
-    "iat": "<issuance date>",
- 		"vc": {
-        <json of the verifiable credential>
-    }
-},
-"signature":
-```
-
-Date and time values MUST be a number containing a NumericDate value (Unix epoch timestamps).
-
-
-
-## 7. Realization of the status registry
-
-The verifiable credential SHOULD have a property for the discovery of information about the current status of a verifiable credential. The property is specified in the [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/#status).
-
-Allowed status of a verifiable credential is valid, suspended, or revoked.
-
-```json
-  "credentialStatus": {
-    "id": "https://example.com/status/<status list identifier>",
-    "type": "CredentialStatusList2017"
-  }
-```
-
-The credentials list is specified by [Credential Status List (2017)](https://w3c-ccg.github.io/vc-csl2017/).
+## 3. Requirements for verifiable credentials
 
+The Nuts framework MUST support the use of verifiable credentials in the roles of Issuer, Holder, and Verifier as described in [Verifiable Credentials Use Cases](https://www.w3.org/TR/vc-use-cases/) and [W3C Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/). In the terms of decentralized Identifiers, the Nuts framework acts as a software agent of the person or organisation accountable for the role of Issuer, Holder, and Verifier.
 
+The Nuts framework MUST support the use decentralized identifiers as described in https://www.w3.org/TR/did-core/.
 
-## 8. Realization of the issuer trust list
+The Nuts framework MAY have limited support on the following issues:
 
-Downloaded list of trusted issuers.
+- The framework supports the JSON Web Keys (JWK) as key format (verification method type equals `JsonWebKey2020`). See also https://www.w3.org/TR/did-spec-registries/.
+- The framework has limited support for key types (minimal OKP) and algorithms (minimal EdDSA).
+- The framework has limited support for DID methods (only web method), which resolve to a DID document in a verifiable data registry. See https://w3c-ccg.github.io/did-method-web/.
 
+The Nuts framework MUST implement a secure Wallet to store decentralized Identifiers (DID) and their private key, and to store verifiable credentials that the agent holds.
 
+The Nuts framework MAY introduce 'irma' as did method to support the IRMA signing and verification.
```

Comment:
I guess this can be removed and be added as separate RFC or part of a RFC when needed?

Reply:
Agreed.